app/models/journeyDomain/transport/TransportDomain.scala

```diff
@@ -28,10 +28,21 @@ case class TransportDomain(
 
 object TransportDomain {
 
-  implicit val userAnswersReader: UserAnswersReader[TransportDomain] =
+  implicit val userAnswersReader: UserAnswersReader[TransportDomain] = {
```

Comment:
Do we need any of the commented out code?

Reply:
Will probably be used for the nav in the next sprint but I didn't get all of it working so I'll just remove it.